mlir/test/Dialect/Vector/vector-multi-reduction-inner-reduction-pass-lowering.mlir

```diff
@@ -0,0 +1,34 @@
+// RUN: mlir-opt -lower-vector-multi-reduction="lowering-strategy=inner-reduction" -split-input-file %s | FileCheck %s
```

Comment:
Can we merge two test files? We can add `--check-prefix=CHECK-RED` to FileCheck for this case.

Reply:
Thank you, the two test files have been combined into a single file.